index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
```

Comment:
Remember the indentation should be 2. Check the [style](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#indentation) guide